src/Telegram/Types/Reaction/ReactionTypePaid.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace SergiX44\Nutgram\Telegram\Types\Reaction;
+
+use JsonSerializable;
+use SergiX44\Hydrator\Resolver\EnumOrScalar;
+use SergiX44\Nutgram\Telegram\Properties\ReactionTypeType;
+use function SergiX44\Nutgram\Support\array_filter_null;
+
+/**
+ * The reaction is paid.
+ * @see https://core.telegram.org/bots/api#reactiontypepaid
+ */
+class ReactionTypePaid extends ReactionType implements JsonSerializable
+{
+    /**
+     * Type of the reaction, always “paid”
+     * @var ReactionTypeType|string
+     */
+    #[EnumOrScalar]
+    public string|ReactionTypeType $type = ReactionTypeType::PAID;
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏enum needs to be the the first type of the union